types/index.d.ts

```diff
@@ -173,7 +173,7 @@ export interface ClientAuthorizationState {
   promptedScopes?: string[];
 }
 
-declare class Interaction {
+export class Interaction extends BaseModel {
```

Comment:
nope, its not an exported Class.

Reply:
How about export it as an interface instead? The reason I need that is to handle interactions under custom routes.

Reply:
nevermind reverted.